community-operators/synopsys/synopsys-operator.2019.1.0.yaml

```diff
@@ -0,0 +1,288 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: synopsys-operator.2019.1.0
+  namespace: placeholder
+  annotations:
```

Comment:
Additional annotations can be found here: https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md

Reply:
fixed in latest push